tests/it/array/utf8/mutable.rs

```diff
@@ -80,3 +80,18 @@ fn test_extend_trusted_len() {
         Some(&Bitmap::from_u8_slice(&[0b00011011], 5))
     );
 }
+
+#[test]
+fn test_extend_values() {
+    let mut array = MutableUtf8Array::<i32>::new();
+
+    array.extend_values([Some("hi"), None, Some("there"), None].iter().flatten());
```

Comment:
why the flatten instead of `[Some("hi"), Some("there")].iter()`?

Reply:
To create an iterator whose length cannot be trusted. The `None` values will be removed.

Reply:
ah, got it! 👍